### PR TITLE
Show the correct operator in plan description

### PR DESCRIPTION
**Pull request by Mats-SX**

### CODE REVIEW

File: manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala [hunk 1]
```diff
@@ -179,8 +179,11 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
       title = "Update Graph",
       text =
         """Applies updates to the graph.""".stripMargin,
-      queryText = """CREATE (:Person {name: "Alistair"})""",
-      assertions = (p) => assertThat(p.executionPlanDescription().toString, containsString("CreateNode"))
+      queryText = """CYPHER planner=rule CREATE (:Person {name: "Alistair"})""",
```
Comment (pontusmelke) [inline manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala:182]: Why do we want examples in the manual with `planner=rule`?

Comment (Mats-SX) [inline manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala:182]: Are you suggesting to remove the section about this operator, `UpdateGraph`?

Comment (pontusmelke) [inline manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala:182]: I think so, I just though we didn't want examples where we use non-default planners and such

Comment (Mats-SX) [inline manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala:182]: Perhaps you're right. There's a card on the Cypher wall asking the team to update the manual with all the new writing operators, but it's not going to be played any time soon. When someone gets around to doing that, they would possibly remove this section. In the meantime, I think making this example anything but nonsensical would be an improvement.

Comment (pontusmelke) [inline manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala:182]: Ok, sounds good.

**Pull request merged**